### PR TITLE
Adding H2O-ai

### PR DESCRIPTION
**Pull request by maheshakya**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,6 +60,7 @@
         <module>bsf/3.0.0.wso2v4</module>
         <module>lucene/4.10.3-wso2v1</module>
         <module>rabbit-mq/3.4.4.wso2v1</module>
+        <module>h2o/3.2.0.wso2v1</module>
```
Comment (thusithathilina) [inline pom.xml]: No need to add to parent pom.

Comment (maheshakya) [inline pom.xml]: Removed.

File: h2o/3.2.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <h2o.version>3.2.0.9</h2o.version>
+        <google-analytics-java.version>1.1.2-H2O-CUSTOM</google-analytics-java.version>
+        <orbit.version.h2o>3.2.0.wso2v1</orbit.version.h2o>
+        <h2o.import.version>[3.2.0.9, 3.6.0.3)</h2o.import.version>
```
Comment (callkalpa) [inline h2o/3.2.0.wso2v1/pom.xml]: This property is not being used

File: h2o/3.2.0.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    </properties>
+
+    <repositories>
```
Comment (callkalpa) [inline h2o/3.2.0.wso2v1/pom.xml]: I don't think we need repositories section

File: h2o/3.2.0.wso2v1/pom.xml (hunk 2 above)
Comment (maheshakya) [inline h2o/3.2.0.wso2v1/pom.xml]: Removed the property.

File: h2o/3.2.0.wso2v1/pom.xml (hunk 3 above)
Comment (maheshakya) [inline h2o/3.2.0.wso2v1/pom.xml]: removed repositories.

File: h2o/3.2.0.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                            !org.ietf.jgss;resolution:=optional,
+                            !org.joda.convert;resolution:=optional,
+                            !org.mortbay.*;resolution:=optional,
+                            !org.slf4j.*;resolution:=optional,
+                            !org.w3c.dom;resolution:=optional,
+                            !org.xml.sax.*;resolution:=optional,
+                            !javassist.bytecode.*;resolution:=optional,
+                            !javassist.expr;resolution:=optional,
+                            !sun.plugin.protocol;resolution:=optional,
+                            !sun.plugin.protocol;resolution:=optional
```
Comment (Nipuni) [inline h2o/3.2.0.wso2v1/pom.xml]: Define the import versions

Comment (maheshakya) [inline h2o/3.2.0.wso2v1/pom.xml]: Do unimports also require versions?
I haven't seen that being practiced.

**Pull request merged**